charts/argo-rollouts/templates/argo-rollouts-service-monitor.yaml

```diff
@@ -0,0 +1,21 @@
+{{- if .Values.controller.metrics.serviceMonitor.enabled }}
+apiVersion: monitoring.coreos.com/v1
+kind: ServiceMonitor
+metadata:
+  name: {{ .Release.Name }}
+  labels:
```

Comment:
I think we may want to allow users to add their own labels and annotations to this template.

Reply:
Added both label and annotation. But I wonder is there really have use case for add annotation...